statics/js/components/topology.js

```diff
@@ -309,6 +332,31 @@ var TopologyComponent = {
         }
     },
 
+    highlightSelectedNodes: function(bool) {
+      var self = this;
+
+      this.$topologyQuery(this.topologySelect)
+        .then(function(nodes) {
+          nodes.forEach(function(n) {
+            for (var i in n.Nodes) {
+                var myNode = n.Nodes[i];
+                if (bool) {
+                    self.$store.commit("highlight", myNode.ID);
```

Comment:
you should use the self.layout.highlightNodeID function as it is the layout which has to handle this.

Reply:
sorry for a stupid question - do you mean without $store? I thought I want to do it asynchronously. Actually copied the code from capturing.

Reply:
There is a dedicated function here: 
https://github.com/skydive-project/skydive/blob/master/statics/js/components/topology.js#L164

I think we need to change the capture stuff but that another story. So if you wanted to stay with your implementation that is OK for me I will change it in my "sliding panel" patch.

Reply:
I will change it

Reply:
It looks like it has bug.. I will check it - the subscribe function sends to highlightNodeID one parameter, while highlightNodeID in layout receives two parameters and doesnt actually use the second one...
Maybe my misunderstanding of JS :(